tests/unit/test_neptune.py

```diff
@@ -160,9 +160,14 @@ def test_sparql_query(neptune_endpoint, neptune_port) -> Dict[str, Any]:
     df = wr.neptune.execute_sparql(client, "INSERT DATA { <test1> <test1> <test1>}")
     df = wr.neptune.execute_sparql(client, "SELECT ?s ?p ?o {?s ?p ?o} LIMIT 1")
     assert df.shape == (1, 3)
+    for col in ["s", "p", "o"]:
+        assert col in df.columns
 
     df = wr.neptune.execute_sparql(client, "SELECT ?s ?p ?o {?s ?p ?o} LIMIT 2")
     assert df.shape == (2, 3)
+    
+    df = wr.neptune.execute_sparql(client, "SELECT ?s ?p ?o {?s ?p ?o} LIMIT 0")
+    assert df.shape == (0, 3)
```

Comment:
Do we want to add a check for the column names here as well? Since the main issue was that the column names wouldn't show up for empty frames.